autoware_launch/config/control/trajectory_follower/lateral/pure_pursuit.param.yaml

```diff
@@ -1,7 +1,16 @@
 /**:
   ros__parameters:
-    # -- algorithm --
-    lookahead_distance_ratio: 2.2
-    min_lookahead_distance: 2.5
-    reverse_min_lookahead_distance: 7.0
+    ld_velocity_ratio: 2.4
+    ld_lateral_error_ratio: 3.6
+    ld_curvature_ratio: 120.0
+    long_ld_lateral_error_threshold: 0.5
+    min_lookahead_distance: 4.35
+    max_lookahead_distance: 15.0
     converged_steer_rad: 0.1
+    reverse_min_lookahead_distance: 7.0
+    prediction_ds: 0.3
+    prediction_distance_length: 21.0
+    resampling_ds: 0.1
+    curvature_calculation_distance: 4.0
+    enable_path_smoothing: false
+    path_filter_moving_ave_num: 25
```

Comment:
It seems `lookahead_distance_ratio` is missing.
Also could you make a PR to `pure_pursuit` package in autoware.universe.

Reply:
It is not missed. Because `lookahead_distance_ratio` is old parameter, I didn't add it. (I guess, we do not use the `pure_pursuit_node.cpp`, it is deprecated.)  

Also, I will create PR in autoware.universe to update its parameters in package.